.plzconfig

```diff
@@ -83,8 +83,9 @@ accept = MPL-2.0
 accept = LGPL
 accept = Artistic License
 accept = ISC
-# Not really a licence, but Bazel projects commonly describe things this way.
+# Not really a license, but Bazel projects commonly describe things this way.
```

Comment:
This was correct before - in British English the noun is `licence`.

(It's very common to see `license` because it's more American-dominated where the noun and verb are the same. I think we aim to accept both in BUILD files.)

Reply:
Ah fair enough 🤣
I got confused because all the licence names above are spelt with an 's'